survival_guide/1_concepts.adoc

```diff
@@ -23,7 +23,17 @@ This topic covers some of the most important concepts of the Fusion cluster depi
 
 //tag::which[]
 
-Although Figure 1 above depicts running in Google Kubernetes Engine (GKE), Fusion 5 runs on any modern version of Kuberentes (v 1.14 or newer). In fact, very little code in Fusion has any awareness of Kubernetes or Docker. The K8s ecosystem is moving very fast and infrastructure providers are quickly offering some flavor of Kubernetes. However, the beauty of Kubernetes is that applications like Fusion do not need to care about the underlying infrastructure. Although Lucidworks provides setup scripts for GKE, AKS, and EKS to help users get started with Fusion 5, do not take this to mean Fusion only runs on those flavors of Kubernetes. In general, as long as K8s is running, Fusion 5 can run on it.
+Figure 1 above depicts Fusion running in Google Kubernetes Engine (GKE). To run Fusion 5, use your cloud provider's stable release channel.
+
+NOTE: GKE refers to the version as *stable release channel*. Other providers may use a different term for the version on which to run Fusion 5. 
+
+For more information, see:
+
+* link:https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels[GKE Release channels]
+* link:https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar[Amazon EKS Kubernetes release calendar]
+* link:https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar[AKS Kubernetes Release Calendar]
+
+Lucidworks provides setup scripts for GKE, AKS, and EKS to help users get started with Fusion 5, but it may be possible to use other flavors of Kubernetes as well. In general, as long as Kubernetes is running, Fusion 5 can run on it.
```

Comment:
This is a nitpick, but we should list the 3 kube platforms in the same order.  The bullets above are GKE, EKS, AKS, but then the sentence below says "GKE, AKS, and EKS".

Reply:
Changed.